index.bs

```diff
@@ -525,7 +527,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 6. Initialize |issuedRequests| to an empty list.
 
-7. For each authenticator currently available on this platform, perform the following steps:
+7. If no authenticator is available, reject |promise| with a <a>DOMException</a> whose name is "NotFoundError", and 
+    terminate this algorithm.
+
+8. For each authenticator currently available on this platform, perform the following steps:
```

Comment:
@jyasskin do you know any useful Bikeshed tricks for referring back to previous steps in an algorithm? Something like "use the quantity computed in step 7" for example is a pain because you have to manually maintain the correctness of the number over time.